build/psr12/convert_pull_requests.php

```diff
@@ -0,0 +1,270 @@
+    $cmd = $gh . ' pr list --limit 1 --json ' . implode(',', $fieldList) . ' --search "is:pr is:open -label:psr12 ' . $branches . '"';
+}
+
+$output = [];
+exec($cmd, $output, $result);
+if ($result !== 0) {
+    var_dump([$cmd, $output, $result]);
+    die('Unable to retrieve PR list.');
+}
+
+$json = $output[0];
+
+if (!empty($prNumber)) {
+    $json = '[' . $json . ']';
+}
+
+$list = json_decode($json, true);
+
+echo "\nFound " . count($list) . " pull request(s).\n";
+
+foreach ($list as $pr) {
+    echo "Checkout #" . $pr['number'] . "\n";
+
+    $cmd    = $gh . ' pr checkout ' . $pr['url'] . ' --force -b psr12/merge/' . $pr['number'];
+    $output = [];
+    exec($cmd, $output, $result);
+    if ($result !== 0) {
+        var_dump([$cmd, $output, $result]);
+        die('Unable to checkout pr #' . $pr['number']);
+    }
+
+    echo "Upmerge to psr12anchor\n";
+
+    $cmd    = $git . ' merge psr12anchor';
+    $output = [];
+    exec($cmd, $output, $result);
+    if ($result !== 0) {
+        var_dump([$cmd, $output, $result]);
+        die('Unable to upmerge to psr12anchor pr #' . $pr['number']);
+    }
+
+    echo "Run PSR-12 converter script\n";
+
+    $cmd = $php . ' ' . $scriptRoot . '/psr12_converter.php --task=branch --repo="' . $repoRoot . '"';
+
+    passthru($cmd, $result);
+    if ($result !== 0) {
+        var_dump([$cmd, $result]);
+        die('Unable to convert to psr-12 pr #' . $pr['number']);
+    }
+
+    echo "Upmerge to psr12final\n";
+
+    $cmd    = $git . ' merge -s ours psr12final';
+    $output = [];
+    exec($cmd, $output, $result);
+    if ($result !== 0) {
+        var_dump([$cmd, $result]);
+        die('Unable to upmerge to psr-12 pr #' . $pr['number']);
+    }
+
+    if (!$createPullRequest && $pr['maintainerCanModify'] === true) {
+        echo "Push directly to PR branch\n";
+
+        $cmd    = $git . ' push git@github.com:' . $pr['headRepositoryOwner']['login'] . '/' . $pr['headRepository']['name'] . '.git '
+            . 'psr12/merge/' . $pr['number'] . ':' . $pr['headRefName'];
+        $output = [];
+        exec($cmd, $output, $result);
+        if ($result !== 0) {
+            var_dump([$cmd, $output, $result]);
+            die('Unable to directly push for pr #' . $pr['number']);
+        }
+
+        $cmd    = $gh . ' pr comment ' . $pr['url'] . ' --body "This pull requests has automatically converted to the PSR-12 coding standard."';
+        $output = [];
+        exec($cmd, $output, $result);
+        if ($result !== 0) {
+            var_dump([$cmd, $output, $result]);
+            die('Unable to pull request for pr #' . $pr['number']);
+        }
+    } else {
+        echo "Create pull request\n";
+
+        $cmd    = $git . ' push --force -u github HEAD';
+        $output = [];
+        exec($cmd, $output, $result);
+        if ($result !== 0) {
+            var_dump([$cmd, $output, $result]);
+            die('Unable to push for pr #' . $pr['number']);
```

Comment:
```suggestion
            die('Unable to directly push for pr #' . $pr['number']);
```

I assume this should be the same message as line 219

Reply:
actually it's not directly to the PR it's to "my github Repository" changed it to 'Unable to push to github for pr #'